Resources/Prototypes/_White/Aspects/Aspects.yml

```
@@ -88,3 +88,104 @@
+    - Загул
+    - Учёт
+    - Разбор
+    - Карман
+    - Обед
+    - Мешок
+    - Полёт
+    - Барак
+    - Украл
+    - Друган
+    lastNames:
+    - Помоев
```

yamllint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)